src/GT.UI/Controllers/JobListingController.cs

```diff
@@ -10,32 +13,47 @@ namespace GT.UI.Controllers
 	public class JobListingController : ControllerBase
 	{
 		private readonly ILogger<JobListingController> _logger;
-		private readonly IGTListingService _listingService;
+		private readonly GTAppContext _listingService;
 
-		public JobListingController(ILogger<JobListingController> logger, IGTListingService listingService)
+		public JobListingController(ILogger<JobListingController> logger, GTAppContext listingService)
 		{
 			_logger = logger;
 			_listingService = listingService;
 		}
 
+
 		// GET: api/<JobListingController>
 		[HttpGet]
-		public IEnumerable<string> Get()
+		public async Task<ActionResult<IEnumerable<Listing>>> GetListing()
 		{
-			return new string[] { "value1", "value2" };
+			var listings = await _listingService.Listings.FindAsync();
```

Comment:
FindAsync returns a single listing, check if this should make use of _listingService.Get() instead.

Reply:
Updated to work with DTO n servicelayers